### PR TITLE
Passing upgrade params only if the env is set.

### PR DESCRIPTION
**Pull request by piyush-nimbalkar**
- Allowing focus param, to selectively run tests

### CODE REVIEW

File: deployments/deploy-aws.sh [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+if [ -n "${FOCUS_TESTS}" ]; then
+    EXTRA_ARGS="--focus=$FOCUS_TESTS $EXTRA_ARGS"
+fi
+
+if [ -n "${STORAGE_UPGRADE_VERSION}" ]; then
+    UPGRADE_ARGS="--storage-driver-upgrade-version=$STORAGE_UPGRADE_VERSION"
+fi
+
+if [ -n "${STORAGE_BASE_VERSION}" ]; then
+    UPGRADE_ARGS="--storage-drvier-base-version=$STORAGE_BASE_VERSION $UPGRADE_ARGS"
```
Comment (adityadani) [inline deployments/deploy-aws.sh]: --storage-**driver**-base-version

File: deployments/deploy-ssh.sh [hunk 2]
```diff
@@ -14,6 +15,18 @@ if [ -n "${SKIP_TESTS}" ]; then
...
+if [ -n "${FOCUS_TESTS}" ]; then
+    EXTRA_ARGS="--focus=$FOCUS_TESTS $EXTRA_ARGS"
+fi
+
+if [ -n "${STORAGE_UPGRADE_VERSION}" ]; then
+    UPGRADE_ARGS="--storage-driver-upgrade-version=$STORAGE_UPGRADE_VERSION"
+fi
+
+if [ -n "${STORAGE_BASE_VERSION}" ]; then
+    UPGRADE_ARGS="--storage-drvier-base-version=$STORAGE_BASE_VERSION $UPGRADE_ARGS"
```
Comment (adityadani) [inline deployments/deploy-ssh.sh]: --storage-**driver**-base-version

**Comment by piyush-nimbalkar** [comment]
Created issue to track the documentation: https://github.com/portworx/torpedo/issues/81

**Comment by piyush-nimbalkar** [comment]
Verified this in jenkins. Merging in master

**Pull request merged**